infra/profile.py

```diff
@@ -1,51 +1,62 @@
 import os
 from glob import glob
 from sys import stdout
 import subprocess
 
-profiles = (
-  glob("benchmarks/passing/**/*.bril", recursive=True)
-)
 
 modes = [
-  # (name, runmode, options)
-  ("rvsdg_roundtrip", "rvsdg-round-trip-to-executable", ""),
-
-  ("egglog_noopt_brilift_noopt", "compile-brilift", "--optimize-egglog false --optimize-brilift false"),
-  ("egglog_noopt_brilift_opt", "compile-brilift", "--optimize-egglog false --optimize-brilift true"),
-  ("egglog_opt_brilift_noopt", "compile-brilift", "--optimize-egglog true --optimize-brilift false"),
-  ("egglog_opt_brilift_opt", "compile-brilift", "--optimize-egglog true --optimize-brilift true"),
-
-  ("egglog_noopt_bril_llvm_noopt", "compile-bril-llvm", "--optimize-egglog false --optimize-bril-llvm false"),
-  ("egglog_noopt_bril_llvm_opt", "compile-bril-llvm", "--optimize-egglog false --optimize-bril-llvm true"),
-  ("egglog_opt_bril_llvm_noopt", "compile-bril-llvm", "--optimize-egglog true --optimize-bril-llvm false"),
-  ("egglog_opt_bril_llvm_opt", "compile-bril-llvm", "--optimize-egglog true --optimize-bril-llvm true")
+    # (name, runmode, options)
+    ("rvsdg_roundtrip", "rvsdg-round-trip-to-executable", ""),
+
+    ("egglog_noopt_brilift_noopt", "compile-brilift",
+        "--optimize-egglog false --optimize-brilift false"),
+    ("egglog_noopt_brilift_opt", "compile-brilift",
+        "--optimize-egglog false --optimize-brilift true"),
+    ("egglog_opt_brilift_noopt", "compile-brilift",
+        "--optimize-egglog true --optimize-brilift false"),
+    ("egglog_opt_brilift_opt", "compile-brilift",
+        "--optimize-egglog true --optimize-brilift true"),
+
+    ("egglog_noopt_bril_llvm_noopt", "compile-bril-llvm",
+        "--optimize-egglog false --optimize-bril-llvm false"),
+    ("egglog_noopt_bril_llvm_opt", "compile-bril-llvm",
+        "--optimize-egglog false --optimize-bril-llvm true"),
+    ("egglog_opt_bril_llvm_noopt", "compile-bril-llvm",
+        "--optimize-egglog true --optimize-bril-llvm false"),
+    ("egglog_opt_bril_llvm_opt", "compile-bril-llvm",
+        "--optimize-egglog true --optimize-bril-llvm true")
 ]
 
+
 def bench(profile):
-  # strip the path to just the file name
-  profile_file = profile.split("/")[-1]
-
-  # strip off the .bril to get just the profile name
-  profile_name = profile_file[:-len(".bril")]
-
-  profile_dir = f'./tmp/bench/{profile_name}'
-  try:
-    os.mkdir(profile_dir)
-  except FileExistsError:
-    print(f'{profile_dir} exists, overwriting contents')
-    
-  for mode in modes:
-    (name, runmode, options) = mode
-    subprocess.call(f'cargo run --release {profile} --run-mode {runmode} {options} -o {profile_dir}/{name}', shell=True)
-
-    with open(f'{profile_dir}/{name}-args') as f:
-      args = f.read().rstrip()
-    
-    subprocess.call(f'hyperfine --warmup 2 --max-runs 100 --export-json {profile_dir}/{name}.json "{profile_dir}/{name} {args}"', shell=True)
+    # strip the path to just the file name
```

Comment:
Are there changes here or just whitespace?

Reply:
It looks like Oliver's editor turned our two spaces into 4 :(

Reply:
@oflatt It would be good to avoid thrashing the diff with whitespace changes. Can you please revert auto-formatting changes? If you want, we should enforce formatting rules with a linter or something, but until it's enforced, we shouldn't commit changes like this since it makes the history harder to look through.

Reply:
yes, sorry I didn't realize I had formatting enabled.